### PR TITLE
Merge up asset-admin test into 3.2

### PR DESCRIPTION
**Pull request by maxime-rainville**
Follow up from https://github.com/silverstripe/silverstripe-graphql/pull/249

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,37 +2,62 @@ language: php
 
 dist: trusty
 
+before_install:
+  - if [[ $BEHAT_TEST ]]; then sudo apt-get update && sudo apt-get install chromium-chromedriver; fi
+
 env:
   global:
-    - COMPOSER_ROOT_VERSION=3.x-dev
+    - COMPOSER_ROOT_VERSION=3.1.x-dev
```
Comment (emteknetnz) [inline .travis.yml]: this should be 3.2.x-dev?

File: .travis.yml [hunk 2]
```diff
@@ -2,37 +2,62 @@ language: php
...
+    - DISPLAY=":99"
+    - XVFBARGS=":99 -ac -screen 0 1024x768x16"
+    - SS_BASE_URL="http://localhost:8080/"
+    - SS_ENVIRONMENT_TYPE="dev"
+    - BEHAT_RECIPE_CMS_VERSION=4.4.x-dev
```
Comment (emteknetnz) [inline .travis.yml]: should this be 4.4 or 4.5?

File: .travis.yml [hunk 3]
```diff
@@ -2,37 +2,62 @@ language: php
...
+    - ASSET_ADMIN_VERSION=1.4.x-dev
```
Comment (emteknetnz) [inline .travis.yml]: should this be 1.4 or 1.5?

File: .travis.yml (hunk 1 above)
Comment (maxime-rainville) [reply .travis.yml]: Good catch, I've incremented to 3.2.x-dev

File: .travis.yml (hunk 2 above)
Comment (maxime-rainville) [reply .travis.yml]: You were looking at the older commit that was targeting the 3.1 branch I think.

File: .travis.yml (hunk 3 above)
Comment (maxime-rainville) [reply .travis.yml]: You were looking at the older commit that was targeting the 3.1 branch I think.

File: .travis.yml [hunk 4]
```diff
@@ -2,37 +2,61 @@ language: php
...
+
 env:
   global:
-    - COMPOSER_ROOT_VERSION=3.x-dev
+    - COMPOSER_ROOT_VERSION=3.2.x-dev
+    - DISPLAY=":99"
+    - XVFBARGS=":99 -ac -screen 0 1024x768x16"
+    - SS_BASE_URL="http://localhost:8080/"
+    - SS_ENVIRONMENT_TYPE="dev"
+    - RECIPE_CMS_VERSION=4.5.x-dev
```
Comment (maxime-rainville) [inline .travis.yml]: The asset-admin unit test need a the full recipe-cms on 4.5.x

**Comment by maxime-rainville** [comment]
Grrrr !!! Garion merge is 7.4 test before I managed to sneak this one in.

**Pull request merged**